protoquant/src/triton/quant.py

```diff
@@ -104,28 +104,29 @@ def quant_kernel(inputs):
         zero_point_from_min_error < zero_point_from_max_error,
```

Comment:
@nmacchioni - I'm also wondering whether this is a decision we could make statically even if it results in a loss of accuracy? Seems like a lot of preceding code (and reductions) depend on this.

Reply:
We could either do zero point from min or max, and just choose which one to use, or we could take zero point as the midpoint of the min and max. Both ways should be fine.